spring-boot-project/spring-boot/src/main/java/org/springframework/boot/logging/structured/StructuredLoggingJsonMembersCustomizer.java

```diff
@@ -24,7 +24,8 @@
  * customize {@link JsonWriter} {@link Members}.
  * <p>
  * An implementation may be provided using the {@code logging.structured.json.customizer}
- * property.
+ * property. Alternatively, implementations may be listed in a
+ * {@code META-INF/spring.factories} file.
```

Comment:
Thanks, @quaff. It'd be nice to align the wording with other classes like `DatabaseInitializerDetector`. Something like:

```
Alternatively, implementations can be registered in {@code META-INF/spring.factories} under the key {@code org.springframework.boot.logging.structured.StructuredLoggingJsonMembersCustomizer}.
```

Reply:
done.